### ISSUE TITLE
Fix documenting version numbers

### ISSUE THREAD

**Issue by nden** [STScI-JWST/jwst#57]
`stpipe.Step.run` records the version of the software used as  `__svn_revision__`.
@jhunkeler What is the way to get the equivalent number now? 

I'm commenting out this line in stpipe for now to try to get a clean run of all tests.

**Comment by nden** [comment]
[this line](https://github.com/STScI-JWST/jwst/blob/master/jwst/stpipe/step.py#L400) and the one below

**Comment by jhunkeler** [comment]
At build-time the version data is recorded at the top-level of the jwst 
package:

`from jwst.version import *`
`from .version import *`
`from ..version import *` #(or whatever level of relative '.'s)

`__version_commit__` replaces `__svn_revision__`


**version.py snippet:**

```
__version__ = '0.6.0noop.dev107'
__version_short__ = '0.6.0noop'
__version_long__ = '0.6.0noop-107-gf845387d'
__version_post__ = '107'
__version_commit__ = 'f845387d'
__version_date__ = '2016-06-13 17:35:10 -0400'
__version_dirty__ = False
__build_date__ = '2016-06-13'
__build_time__ = '17:37:19.913442'
__build_status__ = 'release' if not int(__version_post__) > 0 \
     and not __version_dirty__ \
     else 'development'
```

**Comment by jhunkeler** [comment]
^ replying by email breaks markdown. Good to know.

**Comment by nden** [comment]
Thanks. It shoul dbe resolved by #58

**Comment by hbushouse** [comment]
__version_commit__ certainly seems to be the closest thing to the equivalent of the SVN revision number, so that seems appropriate to use in place of __svn_revision__. Just curious, what does the date/time value in __version_date__ refer to? Is it the time at which the commit was done?

**Comment by jhunkeler** [comment]
@hbushouse Yes, `__version_date__` refers to the timestamp of the commit. For example, if you were to check out the parent tag using `git checkout 0.6.0noop` and run the setup script, the generated `version.py` will read:

```
__version__ = '0.6.0noop'
__version_short__ = '0.6.0noop'
__version_long__ = '0.6.0noop-0-g5915b0e9'
__version_post__ = '0'
__version_commit__ = '5915b0e9'
__version_date__ = '2016-06-08 14:23:41 -0400'
__version_dirty__ = False
__build_date__ = '2016-06-13'
__build_time__ = '22:58:53.064910'
__build_status__ = 'release' if not int(__version_post__) > 0 \
    and not __version_dirty__ \
    else 'development'
```

**Comment by philhodge** [comment]
What are we going to do about the CAL_SVN keyword?  Pretend the "SVN" doesn't mean anything?  Delete it?  Rename it?

**Comment by hbushouse** [comment]
I'm thinking of renaming the keyword to something like "CAL_GIT" (CAL_HASH, CALCOMMT?) and populate it with the value of ```__version_commit__```.

**Comment by philhodge** [comment]
On 06/14/2016 08:35 AM, Howard Bushouse wrote:
> I'm thinking of renaming the keyword to something like "CAL_GIT" 
> (CAL_HASH, CALCOMMT?) and populate it with the value of 
> |__version_commit__|

How about CAL_VCS?  If we wrote a keyword VERSION_COMMIT to the FITS 
header, maybe nobody would notice.

Phil

**Comment by hbushouse** [comment]
I agree that something like VERSION_COMMIT (which we can't use anyway, because it's too many characters) or GITCOMMIT etc. is too generic, because there are also keywords that give the version numbers of things like CRDS and SDP. So we do need to classify it under the "CAL" hierarchy somehow. What would "VCS" in "CAL_VCS" stand for?

**Comment by philhodge** [comment]
On 06/14/2016 08:51 AM, Howard Bushouse wrote:
> I agree that something like VERSION_COMMIT (which we can't use anyway, 
> because it's too many characters) or GITCOMMIT etc. is too generic, 
> because there are also keywords that give the version numbers of 
> things like CRDS and SDP. So we do need to classify it under the "CAL" 
> hierarchy somehow. What would "VCS" in "CAL_VCS" stand for?

Version control system.  I didn't think it was a good idea to tie a 
keyword to a particular VCS.  If we'll have lots of different version 
keywords, we could use the comment section to explains what it refers to.

**Comment by nden** [comment]
I am not sure what we talked about here was resolved except the original question.
I think it would be good to have a description comment in the file for each of the variables.

Other than that I have no further comments. @hbushouse feel free to close if you think this has been resolved.

**Comment by hbushouse** [comment]
Fixed by #1388
